staging/src/k8s.io/cli-runtime/pkg/resource/visitor_test.go

```diff
@@ -197,3 +197,117 @@ func TestFlattenListVisitorWithVisitorError(t *testing.T) {
 		t.Fatal(spew.Sdump(test.Infos))
 	}
 }
+
+func TestExpandPathsToFileVisitors(t *testing.T) {
```

Comment:
since the file contents aren't important, I'd suggest generating these in a t.TempDir() rather than checking in fixtures

Reply:
is it worth including case differences ("AB.yaml" comes before "aa.yaml", etc) ? is that platform-specific?

Reply:
Great idea bout using tempdir.  I made the change.  I also added AB.yaml and aa.yaml.  I tested on Linux and Windows and it doesn't seem to be platform-specific.  The test passes on both.